packages/apps/human-app/frontend/src/modules/worker/email-verification/components/email-verification-process.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  PageCardError,
+  PageCardLoader,
+} from '@/shared/components/ui/page-card';
+import { getErrorMessageForError } from '@/shared/errors';
+import { useEmailVerification } from '../hooks';
+import { EmailVerificationSuccessMessage } from './email-verification-success-message';
+
+interface EmailVerificationProcessProps {
+  token: string;
+}
+
+export function EmailVerificationProcess({
+  token,
+}: Readonly<EmailVerificationProcessProps>) {
+  const { errorMsg, isEmailVerificationError, isEmailVerificationPending } =
+    useEmailVerification(token);
```

Comment:
`useEmailVerification` logic can be part of this component because nothing specific happens there